### ISSUE TITLE
Could you provide the code of FCOS as aux head?

### ISSUE THREAD

**Issue by Artificial-Inability** [Sense-X/Co-DETR#35]
The implementation of anchor-free aux head is kind of different from those with anchors, could you please provide the code for CoFCOS head? Thanks!

**Comment by TempleX98** [comment]
The FCOS auxiliary head is implemented almost identically to ATSS, with the only difference being that we explicitly assign anchors to the predicted points.
`
coords_wh = [torch.ones_like(mlvl_points)*(2**(lvl+6)) for lvl, mlvl_points in enumerate(points)]
coords_wh = torch.cat(coords_wh, dim=0)
coords = torch.cat((concat_points, coords_wh), dim=-1)
coords = bbox_cxcywh_to_xyxy(coords)
`
